Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM jimmycuadra/rust:1.16.0
+FROM debian:jessie-slim
```

Comment:
https://nickjanetakis.com/blog/alpine-based-docker-images-make-a-difference-in-real-world-apps